### PR TITLE
Bug 1813052: Fix workload detail child overlapping notification drawer

### PR DESCRIPTION
**Pull request by jcaianirh**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1813052

### CODE REVIEW

File: frontend/public/style/_layout.scss [hunk 1]
```diff
@@ -6,6 +6,10 @@ body,
 .pf-c-page__main-section,
 .pf-c-drawer {
   height: 100%; // so .co-p-has-sidebar, .yaml-editor are full height
+
+  &__content {
+      z-index: 1;
```
Comment (benjaminapetersen) [inline frontend/public/style/_layout.scss]: `z-index:2` and `z-index:1` seems a bit tight to me, do we have a pattern for this property or have we not yet needed one?

Comment (sg00dwin) [reply frontend/public/style/_layout.scss]: Yes, maybe use units of 10
```
.co-notification-drawer {
  z-index: 10;
}
```

Comment (rhamilto) [inline frontend/public/style/_layout.scss]: @jcaianirh this fix breaks full screen terminal.  Will open a bugzilla.  cc:  @spadgett

**Pull request merged**